packages/story-editor/src/components/publishModal/mainContent/index.js

```diff
@@ -97,6 +102,7 @@ const MainContent = ({
 export default MainContent;
 
 MainContent.propTypes = {
+  handleReviewChecklist: PropTypes.func,
```

Comment:
💭  should this be required? If it's not required, then I'd assume that the button shouldn't be rendered - since it won't do anything.